loopchain/channel/channel_service.py

```diff
@@ -521,16 +521,16 @@ def __unsubscribe_call_by_rest_stub(self):
             return
 
     def __check_last_block_to_rs(self):
-        last_block = self.__radio_station_stub.call_async("GetLastBlock")
-        if last_block['height'] <= self.__block_manager.get_blockchain().block_height:
-            return
-
-        # RS peer Connection Error
-        self.__unsubscribe_call_by_rest_stub()
-        self.stop_check_last_block_rs_timer()
-        if self.__state_machine.state != "SubscribeNetwork":
-            logging.error(f"changing state to subscribe network...")
-            self.__state_machine.subscribe_network()
+        try:
+            self.__radio_station_stub.call("GetLastBlock")
```

Comment:
`last_block` condition statement has disappeared.

Reply:
The function to check the last block is only used to check that the RS peer is alive. The comparison of the `last_block` is excluded because it is also done by annouce_confirmed_block as well.

Reply:
Then you mean this change does not affect original behaviour, right?

Reply:
Yep:)